project/Settings.scala

```diff
@@ -28,7 +28,7 @@ object Settings {
     val scalaJQuery             = "1.2"
     val scalaJSReactCommon      = "0.3.0"
```

Comment:
I see you have released a 0.3.2 of `scalaJSReactCommon` with `ReactProps`? I think this version bump would need to be in the PR.

Reply:
Good catch. this gets evicted but we should do better than this. I'll update it